Kickstarter-iOS/Views/PledgeScreenCTAContainerView.swift

```diff
@@ -0,0 +1,189 @@
+
+    _ = self.rootStackView
+      |> adaptableStackViewStyle(isAccessibilityCategory)
+  }
+
+  // MARK: - View Model
+
+  override func bindViewModel() {
+    super.bindViewModel()
+  }
+
+  // MARK: - Configuration
+
+  // TODO: Will be addressed in functionality PR
+//  func configureWith(value: ) {
+//  }
+
+  // MARK: Functions
+
+  private func configureSubviews() {
+    _ = (self.rootStackView, self)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    _ = ([self.pledgeCTAButton, self.applePayButton], self.ctaStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = (self.disclaimerLabel, self.disclaimerView)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    _ = ([self.ctaStackView, self.disclaimerView], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    self.pledgeCTAButton.addTarget(
+      self, action: #selector(self.pledgeButtonTapped), for: .touchUpInside
+    )
+  }
+
+  private func setupConstraints() {
+    NSLayoutConstraint.activate([
+      self.pledgeCTAButton.heightAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minHeight),
+      self.pledgeCTAButton.widthAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minWidth),
+      self.applePayButton.heightAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minHeight),
+      self.applePayButton.widthAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minWidth),
+      self.disclaimerView.leadingAnchor.constraint(
+        equalTo: self.disclaimerLabel.leadingAnchor, constant: Styles.grid(3)
+      ),
+      self.disclaimerView.trailingAnchor.constraint(
+        equalTo: self.disclaimerLabel.trailingAnchor, constant: Styles.grid(3)
+      )
+    ])
+  }
+
+  // TODO: Functionality PR
+  @objc func pledgeButtonTapped() {}
+}
+
+// MARK: - Styles
+
+private func adaptableStackViewStyle(_ isAccessibilityCategory: Bool) -> (StackViewStyle) {
+  return { (stackView: UIStackView) in
+    let spacing: CGFloat = (isAccessibilityCategory ? Styles.grid(1) : Styles.grid(1))
+
+    return stackView
+      |> \.axis .~ NSLayoutConstraint.Axis.vertical
+      |> \.isLayoutMarginsRelativeArrangement .~ true
+      |> \.layoutMargins .~ UIEdgeInsets.init(
+        top: Styles.grid(2),
+        left: Styles.grid(3),
+        bottom: Styles.grid(6),
+        right: Styles.grid(3)
+      )
+      |> \.spacing .~ spacing
+  }
+}
+
+private let disclaimerLabelStyle: LabelStyle = { label in
+  label
+    |> \.font .~ .ksr_footnote()
+    |> \.textColor .~ .ksr_text_dark_grey_500
+    |> \.numberOfLines .~ 0
+    |> \.textAlignment .~ .center
+    |> \.text %~ { _ in
+      "By pledging you agree to Kickstarter's Terms of Use, Privacy Policy and Cookie Policy"
+    }
+}
+
+private let ctaStackViewStyle: StackViewStyle = { stackView in
+  stackView
+    |> \.axis .~ .horizontal
+    |> \.distribution .~ .fillEqually
```

Comment:
Oh, I see you are using `fillEqually`. I believe with this distribution the width constraints are ignored anyway.